Mail2Bug/WorkItemManagement/IWorkItemManager.cs

```diff
@@ -21,5 +21,12 @@ public interface IWorkItemManager
         void ModifyWorkItem(int workItemId, string comment, Dictionary<string, string> values);
 
         INameResolver GetNameResolver();
+
+        /// <summary>
+        /// Get work item for the given ID.
+        /// </summary>
+        /// <param name="workItemId">the work item ID</param>
+        /// <returns>work item corresponding to the ID</returns>
+        SimpleWorkItem GetWorkItem(int workItemId);
```

Comment:
I'm not sure SimpleWorkItem is the right terminology, since it doesn't really expose a work-item interface. Instead, it exposes access to work item field values. My suggestion is to rename this to WorkItemFields (and GetWorkItemFields accordingly)